packages/turf-meta/test.ts

```diff
@@ -395,7 +413,7 @@ test("geomEach -- multiGeometryFeature-properties", (t) => {
 
 test("flattenEach -- MultiPoint", (t) => {
   featureAndCollection(multiPt.geometry).forEach((input) => {
-    const output = [];
+    const output: MultiPoint[] = []; // this is actually Point[] but flattenEach's types are bad
```

Comment:
flattenEach and flattenReduce both have pretty gnarly typing bugs that need to be fixed. MultiPoint/LineString/Polygon should all get transformed to non-multi Point/LineString/Polygon.